src/device/rsp/rsp_core.c

```diff
@@ -263,6 +265,9 @@ int write_rsp_regs2(void* opaque, uint32_t address, uint32_t value, uint32_t mas
 void do_SP_Task(struct rsp_core* sp)
 {
     uint32_t save_pc = sp->regs2[SP_PC_REG] & ~0xfff;
+    uint32_t save_regs;
+    if (repeat_rsp_task)
+        sp->r4300->mi.regs[MI_INTR_REG] &= ~(MI_INTR_SP | MI_INTR_DP | MI_INTR_AI | MI_INTR_SI);
```

Comment:
Sorry you can ignore this PR, the code we are talking about is here:

https://github.com/Gillou68310/mupen64plus-core/commit/b75be1febcc25daea4b286edd92457a3b95e7776